easybuild/easyblocks/l/lammps.py

```diff
@@ -121,6 +121,8 @@ def prepare_step(self, *args, **kwargs):
 
     def configure_step(self, **kwargs):
         """Custom configuration procedure for LAMMPS."""
+        if not get_software_root('VTK'):
+            self.cfg['user_packages'] = [x for x in self.cfg['user_packages'] if x != 'VTK']
```

Comment:
This feels unsafe, but I'm not a Python expert.... @boegel ??

Reply:
I think it's safe, in that it's just removing 'VTK' from the list if it's present, and otherwise keeping everything. But happy for @boegel to decide :-)

Reply:
I meant that user_packages is on both sides of the operation...

Reply:
Should be fine I think. It builds the list on the right then assigns it... IIRC.

```
>>> a = [1,2,3]
>>> a = [x for x in a if x > 1]
>>> a
[2, 3]
```

Reply:
It's fine indeed: you're creating a new list first (based on `self.cfg['user_packages']`), and then do the assignment which overwrites the `self.cfg['user_packages']` value.